### PR TITLE
DDT-921 moved carrierBookingReference from cargoItem to shipmentEquip…

### PR DESCRIPTION
**Pull request by pimg**
…ment
resolves: DDT-921

### CODE REVIEW

File: src/test/java/org/dcsa/core/events/service/impl/ShipmentEquipmentServiceImplTest.java [hunk 1]
```diff
@@ -988,20 +1071,8 @@ void testFindShipmentEquipmentWithMultipleCargoItems() {
...
-                        .getCarrierBookingReference());
-                assertEquals(
-                    "CBR2",
-                    shipmentEquipmentTOS
-                        .get(0)
-                        .getCargoItems()
-                        .get(1)
-                        .getCarrierBookingReference());
+                assertEquals("CBR1", shipmentEquipmentTOS.get(0).getCarrierBookingReference());
+                assertEquals("CBR1", shipmentEquipmentTOS.get(0).getCarrierBookingReference());
```
Comment (nt-as) [inline src/test/java/org/dcsa/core/events/service/impl/ShipmentEquipmentServiceImplTest.java:1075]: This looks like 1075 redundant (a copy of the previous line).

File: src/main/java/org/dcsa/core/events/repository/impl/CargoItemCustomRepositoryImpl.java [hunk 2]
```diff
@@ -24,37 +24,40 @@
   private static final Table CARGO_ITEM_TABLE = Table.create("cargo_item");
   private static final Table CARGO_LINE_ITEM_TABLE = Table.create("cargo_line_item");
   private static final Table SHIPMENT_EQUIPMENT_TABLE = Table.create("shipment_equipment");
-  private static final Table SHIPMENT_TABLE = Table.create("shipment");
+  //  private static final Table SHIPMENT_TABLE = Table.create("shipment");
```
Comment (mp-assco) [inline src/main/java/org/dcsa/core/events/repository/impl/CargoItemCustomRepositoryImpl.java:27]: remove commented code?

File: src/test/java/org/dcsa/core/events/service/impl/ShipmentEquipmentServiceImplTest.java (hunk 1 above)
Comment (pimg) [reply src/test/java/org/dcsa/core/events/service/impl/ShipmentEquipmentServiceImplTest.java]: removed

File: src/main/java/org/dcsa/core/events/repository/impl/CargoItemCustomRepositoryImpl.java (hunk 2 above)
Comment (pimg) [reply src/main/java/org/dcsa/core/events/repository/impl/CargoItemCustomRepositoryImpl.java]: removed

**Review by mp-assco — APPROVED** [review]
Looks good to me

**Pull request merged**